ci/jenkins/pipelines/prs/skuba-test.Jenkinsfile

```diff
@@ -49,26 +51,26 @@ pipeline {
         stage('Getting Ready For Cluster Deployment') { steps {
             sh(script: 'make -f skuba/ci/Makefile pre_deployment', label: 'Pre Deployment')
             sh(script: 'make -f skuba/ci/Makefile pr_checks', label: 'PR Checks')
+            sh(script: "pushd skuba; make -f Makefile install; popd", label: 'Build Skuba')
         } }
 
-        stage('Cluster Deployment') { steps {
-            sh(script: 'make -f skuba/ci/Makefile deploy', label: 'Deploy')
-            archiveArtifacts("skuba/ci/infra/${PLATFORM}/terraform.tfstate")
-            archiveArtifacts("skuba/ci/infra/${PLATFORM}/terraform.tfvars.json")
-        } }
+        stage('Run test workers') { steps {
+            sh(script: "make -f skuba/ci/Makefile test_prs", label: "test_prs")
+            sh(script: "make --keep-going -f skuba/ci/Makefile cleanup", label: 'Cleanup')
```

Comment:
If you move the clean up here instead of in the post block and the `test_prs` step fails we're going to end up with leftover clusters.

Reply:
I will add it also in the post section.